src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,121 @@
+package duke.main;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidTaskTypeException;
+import duke.task.TaskManager;
+import duke.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+import java.util.Scanner;
+
+public class Duke {
+    private static String separatingLine = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+
+    public static void printGreetings(String greeting) {
+
+        if (greeting.equals("hello")) {
+            String logo = " ____        _        \n"
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+            System.out.println("Hello from\n" + logo);
+
+            System.out.println(separatingLine);
+
+            System.out.println("Hello! I'm duke.main.Duke\n" + "What do you want to do today?");
+            System.out.println(separatingLine);
+        } else if (greeting.equals("bye")) {
+            System.out.println(separatingLine);
+            System.out.println("Bye. See you tomorrow!");
+            System.out.println(separatingLine);
+        }
+    }
+
+    public static void main(String[] args) throws DukeException, IOException {
+        TaskManager todayList = new TaskManager();
+
+        printGreetings("hello");
+
+        String fileName = "duke.txt";
+        Path filePath = null;
+        FileWriter fw = new FileWriter(String.valueOf(filePath));
+
+        Scanner sc = new Scanner(filePath);
+        filePath = Paths.get(fileName);
+
+//        Scanner sc = new Scanner(System.in);
+          String text = sc.nextLine();
```

Comment:
Should the comment be removed? Also for line 55, should the indentation follow the code on top?